plugins/cue_points/base/CuePointPlugin.php

```diff
@@ -289,7 +289,9 @@ public static function getCuePointSearchData(entry $entry)
 			return;
 		
 		$dataByType = array();
+		CuePointPeer::setUseCriteriaFilter(false);
```

Comment:
This way you may load deleted cue-points, please tag the criterion that you want to disable and use the tag to disable only this one.